### DISCUSSION TITLE
A row disappeared in my data...

### DISCUSSION THREAD

**Discussion by inesmtc** [jspsych/jsPsych#898] [Q&A] [answered]
Hi everyone,

A row seems to disappear from my data... When I run my experiment locally, I can see that the row corresponding to a break is written in my data (using jsPsych.data.get().csv in the browser's console when the break happen):

"44485.92500004452","<p style=""font-size:160%;"">Pause</p><p style=""font-size:120%;""> Vous avez une pause de 2 minutes. </p><p style=""font-size:120%;""> Appuyer sur la barre d’espace pour reprendre maintenant. </p><div style=""width: 600px; height: 20px; background-color: #ccc;""><div style=""width: 0%; height: 20px; background-color: #333; animation-name: progress-bar; animation-duration: 120s; animation-timing-function: linear;""></div><style> @keyframes progress-bar { 0% { width:0%;} 100% { width:100%;}} </style>","","html-keyboard-response","936","811451","0.0-2.0-2.0-12.0-0.205-0.205","Im_21072020_16:58:28","","experiment","pause","","","32","","","","","","","","","","","","","","","","",""

But when I open my downloaded data this row is not written in my csv file... How is it possible? No treatment is done on my csv and I have the same problem using different way to download my data.
Here is the code of the variable corresponding to the break. I should have the element 'pause' in the screenID column but I haven't :/

var pause = {
      data: {
        partID: 'experiment',
        screenID: 'pause',
      },
      type: 'html-keyboard-response',
      stimulus: '<p style="font-size:160%;">Pause</p>'+
      '<p style="font-size:120%;"> Vous avez une pause de 2 minutes. </p>'+
      '<p style="font-size:120%;"> Appuyer sur la barre d’espace pour reprendre maintenant. </p>'+
      '<div style="width: 600px; height: 20px; background-color: #ccc;">' +
      '<div style="width: 0%; height: 20px; background-color: #333; animation-name: progress-bar; animation-duration: 120s; animation-timing-function: linear;">' +
      '</div>' +
      '<style> @keyframes progress-bar { 0% { width:0%;} 100% { width:100%;}} </style>',
      trial_duration: break_time,
      choices: ['space']
    }

Thanks for your help!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @inesmtc,

How are you downloading the CSV file? Are you using `.localSave()`, a server, something else?

**Reply by inesmtc** [reply]
When I run my experiment locally I use this function:

function downloadCSV(csv, filename){
    var csvFile;
    var downloadLink;
    csvFile = new Blob([csv], {type:"text/csv"});
    downloadLink = document.createElement("a");
    downloadLink.download = filename; 
    downloadLink.href = window.URL.createObjectURL(csvFile);
    downloadLink.style.display = 'none';
    document.body.appendChild(downloadLink);
    downloadLink.click();
}

But I also run it with Pavlovia and I have the save problem :/
Moreover I just noticed that the same problem occurs with other rows...  I'm confused.

**Reply by jodeleeuw** [reply]
Is there anything that the missing rows have in common? Same kind of trial type? Use CSS animation? etc.?

**Reply by inesmtc** [reply]
Ok I just found something! The problem comes from the importation of the data in excel... If I open my csv file in excel and THEN separate the text into different columns, some rows disappear. But if I open a new excel tab and THEN import my data as a csv, everything is fine... I do not understand what excel is doing but it seems to be the reason of my problem ahah
